### PR TITLE
[15.0][ADD] fieldservice_sale_reference

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds new field "sale_reference" to fsm.order that autofill with the sales order

### CODE REVIEW

File: fieldservice_sale_reference/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        Adds new field "sale_reference" to fsm.order that autofill with the sales order
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/solvosci/slv-field-service",
+    "depends": [
+        "fieldservice_sale",
+        "sale_order_client_order_ref_mandatory",
```
Comment (dalonsod) [inline fieldservice_sale_reference/__manifest__.py]: This dependency should be removed IMO

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment. When done, I'll test it ASAP

File: fieldservice_sale_reference/__manifest__.py (hunk 1 above)
Comment (ChristianSantamaria) [reply fieldservice_sale_reference/__manifest__.py]: Its true

**Comment by dalonsod** [comment]
@ChristianSantamaria could you test this?

- Create a sales order that implies a Field Service creation from any role (e.g. Mitchel Admin).
- From a FSM user role (a user different than the first one), look for that Field Service, when (1) has no Sales permissions, or (2) has Sale permissions, but only for his own documents (not Mitchell Admin ones, indeed).

I can test if related field could fire a permissions error in any of these cases

**Comment by ChristianSantamaria** [comment]
@dalonsod 
I just tried it and apparently no error occurs in any case that you mention, but it may be good to add that the field is only visible to sales users

**Comment by dalonsod** [comment]
> I just tried it and apparently no error occurs in any case that you mention, but it may be good to add that the field is only visible to sales users

Well, I think this is a non-sensitive but useful data for a FSM User (and no sales order access is provided, anyway), so it's good to be shown for everybody. So LGTM 👍

**Review by dalonsod — APPROVED** [review]
Only two minor changes pending:

- Move to `Field Service` category in manifest, as `fieldservice_sale` has.
- Proper license should be AGPL one, because depends on an addon that belogs to it.

**Pull request merged**